src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -26,7 +25,7 @@ public Movie add(Movie movie) {
             if (transaction != null) {
                 transaction.rollback();
             }
-            throw new DataProcessingException("Can't insert movie " + movie, e);
+            throw new DataProcessingException("Can't insert cinema hall " + movie);
```

Comment:
The error message in the `DataProcessingException` should refer to the correct entity. Change "Can't insert cinema hall" to "Can't insert movie" to accurately reflect the operation being performed.